nrf_security/cmake/psa_crypto_config.cmake

```diff
@@ -97,7 +105,6 @@ kconfig_check_and_set_base_to_one(PSA_CRYPTO_DRIVER_ALG_RSA_PSS_CC3XX)
 kconfig_check_and_set_base_to_one(PSA_CRYPTO_DRIVER_ALG_SHA_1_CC3XX)
 kconfig_check_and_set_base_to_one(PSA_CRYPTO_DRIVER_ALG_SHA_224_CC3XX)
 kconfig_check_and_set_base_to_one(PSA_CRYPTO_DRIVER_ALG_SHA_256_CC3XX)
-kconfig_check_and_set_base_to_one(PSA_CRYPTO_DRIVER_ALG_SHA_384_CC3XX)
```

Comment:
Commit messages does not mention why this was removed.

Is it not in use?

Reply:
Fixed